### PR TITLE
Korean site translation

### PR DESCRIPTION
**Pull request by sbrichards**
- Add Korean translation pages
- Add sitewide "Languages" dropdown menu

Preview:
https://deploy-preview-154--ethereumorg.netlify.com

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -1,28 +1,80 @@
...
+          'Ethereum is a global, decentralized platform for money and new kinds of applications. On Ethereum, you can write code that controls money, and build applications accessible anywhere in the world.'
+      }
+    ],
     ['meta', { property: 'og:url', content: 'https://ethereum.org' }],
-    ['meta', { property: 'og:image', content: 'https://ethereum.org/assets/img/hero.bc77fa26.jpg'}]
+    [
+      'meta',
+      {
+        property: 'og:image',
+        content: 'https://ethereum.org/assets/img/hero.bc77fa26.jpg' // TODO this 404s
```
Comment (sbrichards) [inline docs/.vuepress/config.js]: @alancwoo I noticed this 404s. Given that the new homepage hero is now an mp4, we should add a static image for this `og:image` property, which is used to generate preview images for social shares. Do you have one handy? If not, I could just take a screenshot of the mp4?

File: docs/.vuepress/theme/components/Footer.vue [hunk 2]
```diff
@@ -7,31 +7,11 @@
       <li><a href="https://github.com/ethereum" target="_blank">GitHub</a></li>
       <li><a href="https://twitter.com/ethereum" target="_blank">Twitter</a></li>
       <li><a href="https://blog.ethereum.org/">Blog</a></li>
-      <!-- <li><a href="#">Branding</a></li> -->
       <li><router-link to="/privacy-policy/">Privacy Policy</router-link></li>
       <li><router-link to="/terms-of-use/">Terms of Use</router-link></li>
       <li><router-link to="/cookie-policy/">Cookie Policy</router-link></li>
       <li><a href="mailto:press@ethereum.org">Contact</a></li>
     </ul>
-    <!-- Matomo -->
```
Comment (sbrichards) [inline docs/.vuepress/theme/components/Footer.vue]: This script was was causing errors in my Netlify previews. Added it back here (using the config): https://github.com/ethereum/ethereum-org-website/pull/159

File: docs/.vuepress/theme/components/Navbar.vue [hunk 3]
```diff
@@ -1,125 +0,0 @@
-<template>
```
Comment (sbrichards) [inline docs/.vuepress/theme/components/Navbar.vue]: @alancwoo this file was not being used anywhere. Safe to delete?

**Comment by alancwoo** [comment]
Hi @sbrichards thanks for all this, it's really great! Just a few notes that may require some external involvement:

1. Footer needs translation
2. Dates aren't translated - @jjmstark is this intentional?
3. The phrase 'Updated Often' is not translated
4. Is it possible for the UI to be more aligned with the design/current UI?

image

**Comment by jjmstark** [comment]
@alancwoo 

1, 2 & 3. Intentional - this is the product of a team of Korean-speaking volunteers, who chose which parts should remain in english.
4. Defer to @sbrichards! Is this a fix we can make easily and quickly?

**Pull request merged**